### ISSUE TITLE
About fcn code:

### ISSUE THREAD

**Issue by hhloverr** [yassouali/pytorch_segmentation#69]
Ask a question: In fcn code, 
'''        
adjstd_pool4 = self.adj_pool4(0.01 * pool4)
add_out_pool4 = self.up_pool4_out(adjstd_pool4[:, :, 5: (5 + up_output.size()[2]),
                                          5: (5 + up_output.size()[3])]
                                          + up_output)
'''
Why 0.01 * pool4?

**Comment by yassouali** [comment]
Hi, if I was remembering correctly, it was used to have more stable gradients, but I do think not using it will not change things significantly, this can be removed with a vgg16 with batch norm.

**Comment by hhloverr** [comment]
> Hi, if I was remembering correctly, it was used to have more stable gradients, but I do think not using it will not change things significantly, this can be removed with a vgg16 with batch norm.

Thank you.
